common/genes/wc_gene_facial_markings.txt

```diff
@@ -440,6 +440,149 @@
 				boy = male
```

Comment:
Localized markings in the Ruler Designer and Barbershop.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127224597-e39d91dd-9631-40cf-a6b3-c9b5fc484464.png)

</details>